README.md

```diff
@@ -108,7 +114,12 @@ java -jar target/quarkus-app/quarkus-run.jar \
     -p JBOSS_JIRA_PWD \
     start-release \
```

Comment:
Is `start-release` the correct name ? Should we instead use `prepare-release` or `setup-release` ?

Reply:
I think this will eventually become the command that kicks off the release. ATM we have a previous step executed by the [snowdrop-automation-client](https://github.com/snowdrop/snowdrop-automation-client) but the idea is to migrate this functionality into the release-manager in this step.

The team's feedback would be also good to have on this.

Reply:
Regarding that this command aims to do (if I'm not wrong):
 - Create the new folders **if** a **new major** release will be done. 
 - Align upstream versions from Spring Boot (currently done by the automation client, it will be implemented in the release manager)
 - Create jira tickets to ask for components versions to use (currently done by the `start-release` command)
 - Set up the Cpaas configuration (currently done by the `start-release` command)

I like `setup-release`.

Reply:
+1 on `setup-release`.

Reply:
`setup-release` 👍🏽

Reply:
`setup-release` sounds good